tests/test-map.c

```diff
@@ -67,4 +83,96 @@ void test_map(void) {
 				 "MAP empty checks after operations");
 		cap_map_free(map);
 	}
+	{//Tests on iterators
+
+		cap_map *map = cap_map_init(4,(void *)3);
```

Comment:
This is incorrect. 

The first argument is the size of the key, so if you're using an `int`, the first argument is the `sizeof(int)` and the 2nd one is the function pointer. This code will not compile. You should create a function(or use the existing one) and pass the name of the function as the 2nd argument. 

Like `cap_map_init(sizeof(int), compare_fn_int);`